docker/package/tezos_setup_wizard.py

```diff
@@ -324,15 +328,42 @@ def pprint_options(self):
             pass
 
         if self.options and isinstance(self.options, list):
-            str_format = "{:1}. {}"
+            options_count = 0
             for o in self.options:
-                if def_i is not None and i == def_i:
-                    print(str_format.format(i, "(default) " + o))
+                if isinstance(o, dict):
+                    for k, values in o.items():
```

Comment:
I don't see `k` used anywhere, why not
```suggestion
                    for values in o.values():
```